.github/workflows/ci-pipeline.yml

```
@@ -109,31 +110,46 @@ jobs:
   verification-summary:
     # Ensure all containers passed the verification
     runs-on: ubuntu-latest
-    name: Verification Summary
+    name: Check Matrix Outcome
     needs:
       - get-containers
       - vib-verify
     if: ${{ always() }}
     steps:
-      - name: Verification Summary
-        run: |
-          if [[ "${{ needs.get-containers.result }}" != "success" ]]; then
-            echo "Verification is required."
-            echo "If you've just created this PR, don't worry about this message. Bitnami Team has to review it and make the verification possible."
-            exit 1
-          else
-            if [[ "${{ needs.get-containers.outputs.result }}" == "skip" ]]; then
-              echo "It seems these changes don't involve any container"
-            else
-              if [[ "${{ needs.vib-verify.result }}" != "success" ]]; then
-                echo "Verification is required"
-                echo "Please review previous jobs to get more information"
-                exit 1
-              else
-                echo "Well done! Everything looks good. Please wait for the Bitnami Team review."
-              fi
-            fi
-          fi
+      - name: Check Status
+        uses: actions/github-script@v6
+        with:
+          script: |
+            state = 'success'
+            description = 'Well done! Everything looks good. Please wait for the Bitnami Team review.'
+            if ("${{ needs.get-containers.result }}" != "success" ) {
+              description = "If you've just created this PR, don't worry about this message. Bitnami Team has to review it and make the verification possible."
```

Thanks for the PR @fmulero, it LGTM! Just a minor comment.

```suggestion
              description = "If you've just created this PR, don't worry about this message. The Bitnami Team has to review it and make the verification possible."
```